### ISSUE TITLE
Add VMware Tools

### ISSUE THREAD

**Issue by fnoir** [Ysurac/openmptcprouter#117]
## Expected Behavior

VMware tools enabled

## Actual Behavior

If it possible to add VMware tools by default on VMware image

## Specifications

  - OpenMPTCProuter version: openmptcprouter v0.30 r7201-4f3082583a
  - OpenMPTCProuter platform: VMware ESXi 6.7

**Comment by Ysurac** [comment]
It's difficult to add them by default, because vmware image is created from the raw image.
But I can add them as available package (if I can compile it). It's https://github.com/vmware/open-vm-tools ?

**Comment by fnoir** [comment]
Yes, it's [https://github.com/vmware/open-vm-tools](https://github.com/vmware/open-vm-tools).

if it isn't possible to add them by default on VMware image, add open-vm-tools on available package is a good idea !

**Comment by Ysurac** [comment]
It will be available when I will have a real working package. Should be ok next week.

**Comment by fnoir** [comment]
Thanks

**Comment by fnoir** [comment]
I use open-vm-tools (v10.3.5-1) and 0 problem. Great job!
